src/lib/styles/tools/_caret.scss

```diff
@@ -6,7 +6,7 @@
     height: 0.4375rem;
     border-width: 2px;
     border-style: none solid solid none;
-    border-color: currentColor;
+    border-color: currentcolor;
```

Comment:
@adamkudrna Can you tell us whether to use `currentColor` vs `currentcolor`? I thought that `currentcolor` is CSS thing and  `currentColor` is SCSS thing, but @mbohal changed it across whole RUI to `currentcolor`

Reply:
`currentColor` was causing style lint errors so I changed it according to the suggestion to make errors go away.